test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -73,6 +80,8 @@ public void P2PDifferentFrameworks(DnxSdk sdk)
 
             // Assert
             Assert.Equal(0, result.ExitCode);
+
+            TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
```

Comment:
Then, I think there should be a disposable test scope which would do the cleanup. The tests would use the `using` statement to define the scope. Otherwise you may leave leftovers behind. It is also quite easy to miss adding `TestUtils.CleanUpTestDir<DnuPackTests>(sdk);` or even hard to know when it is needed...